ADMVEDU95/ADMVEDU95/SceneDelegate.swift

```diff
@@ -9,29 +9,35 @@ import UIKit
 import FirebaseAuth
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
-
     var window: UIWindow?
-    var initialVC = UIViewController()
+    let builder = ViewBuilder()
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
 
         guard let windowScene = (scene as? UIWindowScene) else { return }
         self.window = UIWindow(windowScene: windowScene)
         
+        let router: RouterMain?
+        let navigationController = UINavigationController()
+        
         if Auth.auth().currentUser == nil {
```

Comment:
this should not be in sceneDelegate, please move it to a router, e.g. ApplicationRouter